eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -11,7 +11,7 @@ description:
 
 category:           Network
 build-type:         Simple
-data-files:         data/*.json
+data-files:         test/data/*.json
```

Comment:
Oops, sorry about that.  Did it cause issues, or you happened to notice?

Reply:
hydra CI failed, that's how I noticed. No big deal.